manage.py

```diff
@@ -153,13 +153,13 @@ def run_with_test_server(test_command, coverage, accumulate):
         if b' * Running on' in line:
             break
     test_process = subprocess.Popen(test_command)
-    test_return_code = test_process.wait(timeout=60)
+    test_return_code = test_process.wait(timeout=90)
     # Once the test process has completed we can shutdown the server. To do so
     # we have to make a request so that the server process can shut down
     # cleanly, and in particular finalise coverage analysis.
     # We could check the return from this is success.
     requests.post('http://localhost:5000/shutdown')
-    server_return_code = server.wait(timeout=60)
+    server_return_code = server.wait(timeout=90)
```

Comment:
So I actually think the problem was that `run_test_with_server` was trying to post to `http://localhost:5000/shutdown` whereas because `run_test_server` was now using `TESTSERVER_PORT` it was listening on 5001.

Otherwise I find it hard to believe that the casper process would report entire success, unless it happened to finish very very close to the timeout.

However, 60-90 is probably okay. It just means you might have to wait longer for the test process to give up.